src/microbe_stage/BiomeType.cs

```diff
@@ -1,12 +1,7 @@
 ﻿/// <summary>
 ///   This contains predefined biome types
 /// </summary>
-/// <remarks>
-///   <para>
-///     The order of this enum CANNOT be changed. They are used as patch IDs in classic map,
```

Comment:
This comment actually needs to be kept, but reworded to say that the order of the enum values may not be changed due to saves saving the values. So reordering the values will break all existing saves, which is very bad.